### PR TITLE
Add timeouts to kafka-metrics-exporter tests

### PR DESCRIPTION
**Pull request by colin-grapl**
This is a two line PR to add timeouts to the kafka-metrics-exporter tests. We should probably add more timeouts to this crate in general.

### CODE REVIEW

File: etc/sample_data/min_eventlog.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<Event xmlns='http://schemas.microsoft.com/win/2004/08/events/event'><System><Provider Name='Microsoft-Windows-Sysmon' Guid='{5770385F-C22A-43E0-BF4C-06F5698FFBD9}'/><EventID>11</EventID><Version>2</Version><Level>4</Level><Task>11</Task><Opcode>0</Opcode><Keywords>0x8000000000000000</Keywords><TimeCreated SystemTime='2019-07-24T17:57:21.024458900Z'/><EventRecordID>315</EventRecordID><Correlation/><Execution ProcessID='3324' ThreadID='3220'/><Channel>Microsoft-Windows-Sysmon/Operational</Channel><Computer>DESKTOP-FVSHABR</Computer><Security UserID='S-1-5-18'/></System><EventData><Data Name='RuleName'></Data><Data Name='UtcTime'>2019-07-24 17:57:21.020</Data><Data Name='ProcessGuid'>{87E8D3BD-9BF8-5D38-0000-0010173C0F00}</Data><Data Name='ProcessId'>4820</Data><Data Name='Image'>C:\Program Files (x86)\Google\Update\GoogleUpdate.exe</Data><Data Name='TargetFilename'>C:\Program Files (x86)\Google\Update\Download\{8A69D345-D564-463C-AFF1-A69D9E530F96}\75.0.3770.142\75.0.3770.142_chrome_installer.exe</Data><Data Name='CreationUtcTime'>2019-07-24 17:57:21.020</Data></EventData></Event>
```
Comment (jgrillo-grapl) [inline etc/sample_data/min_eventlog.xml:1]: what is this file?

**Review by jgrillo-grapl — APPROVED** [review]
LGTM

**Comment by wimax-grapl** [comment]
kafka-metrics-exporter-integration-tests_1  \| ---- integration_tests::smoketest stdout ----
--
  | kafka-metrics-exporter-integration-tests_1  \| Error: Elapsed(())
  | kafka-metrics-exporter-integration-tests_1  \| thread 'integration_tests::smoketest' panicked at 'assertion failed: `(left == right)`
  | kafka-metrics-exporter-integration-tests_1  \|   left: `1`,
  | kafka-metrics-exporter-integration-tests_1  \|  right: `0`: the test returned a termination value with a non-zero status code (1) which indicates a failure', /rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/test/src/lib.rs:193:5
  | kafka-metrics-exporter-integration-tests_1  \| note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
  | kafka-metrics-exporter-integration-tests_1  \|

**Comment by jgrillo-grapl** [comment]
I'd recommend rebasing this PR, the underlying infrastructure has changed

**Pull request closed without merging**